sudo/settings.py

```diff
@@ -45,3 +45,6 @@
 # The name of the session attribute used to preserve the redirect destination
 # between the original page request and successful sudo login.
 REDIRECT_TO_FIELD_NAME = getattr(settings, 'SUDO_REDIRECT_TO_FIELD_NAME', 'sudo_redirect_to')
+
+# The name of the view function for reversing
+SUDO_VIEW_NAME = getattr(settings, 'SUDO_VIEW_NAME', 'sudo.views.sudo')
```

Comment:
I don't mind that either. I figured this way might work better with getsentry since we have this multi domain stuff going on and I was not sure what happens if you do cross domain but cache the result from reverse.